java/src/com/ibm/streamsx/topology/context/ContextProperties.java

```diff
@@ -105,4 +105,20 @@
      * See {@link com.ibm.streamsx.topology.Topology#createSubmissionParameter(String, Class)}
      */
     String SUBMISSION_PARAMS = "topology.submissionParams";
+
+    /**
+     * Flag to be supplied to force the compilation to occur on a 
+     * Bluemix instance, if possible. Currently only be used in
+     * conjunction with the {@code ANALYTICS_SERVICE} context.
+     * <p>
+     * Its values can be {@code true} and {@code false}. The
+     * former will force remote compilation when possible, while
+     * the latter will force local compilation, if possible.
```

Comment:
I think only when force is `true` should it force anything. When it's not set or `false` it should have no impact.